### PR TITLE
Fix CI, add skipPreflight, update inflation api

### PR DESCRIPTION
**Pull request by jstarry**
#### Changes
- Reverted 2 rogue dependabot commits that didn't pass CI but merged anyways
- Added `skipPreflight` option to transaction sending APIs
- Updated `getInflation` to `getInflationGovernor`
- Use max commitment blockhashes when sending transactions
- Added `getFeeCalculatorForBlockhash` method (needed for a test)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "bpf-sdk:install": "npm run clean:fixtures; bin/bpf-sdk-install.sh .",
     "bpf-sdk:remove-symlinks": "find bpf-sdk -type l -print -exec cp {} {}.tmp \\; -exec mv {}.tmp {} \\;",
     "build": "cross-env NODE_ENV=production rollup -c",
-    "build:fixtures": "./test/fixtures/noop-c/build.sh; ./test/fixtures/noop-rust/build.sh",
+    "build:fixtures": "echo blah",
```
Comment (CriesofCarrots) [inline package.json:43]: I'm guessing you didn't mean to push this change?

Comment (jstarry) [reply package.json:43]: I'm glad I went with blah this time! 😆

Comment (mvines) [reply package.json:43]: 🤬 🧼

Comment (jstarry) [reply package.json]: 🤐 fixed!

**Comment by jstarry** [comment]
@CriesofCarrots CI is fixed now, ready for another review

**Pull request merged**